File: JTS/CF-A/CF431-D2-A/main.cpp
//    Date : 2019-03-13
//  Author : Rahul Sharma
// Problem : http://codeforces.com/contest/431/problem/A

#include <iostream>
#include <string>

int main() {
    using namespace std;

    ios_base::sync_with_stdio(false);
    cin.tie(0);

    int a[4];
    for(auto& x: a)
        cin >> x;

    string s;
    cin.ignore();
    getline(cin, s);

    int cal = 0;
    for(auto c: s)
        cal += a[c - '1'];

    cout << cal << '\n';
}

File: JTS/CF-A/CF431-D2-A/Main.java
//    Date : 2019-03-20
//  Author : Rahul Sharma
// Problem : http://codeforces.com/contest/431/problem/A

import java.util.*;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int[] count = new int[4];
        for (int i = 0; i < 4; i++)
            count[i] = sc.nextInt();

        sc.nextLine();
        String s = sc.nextLine();
        int calories = 0;
        for (int i = 0; i < s.length(); i++)
            calories += count[s.charAt(i) - '1'];
        System.out.println(calories);
    }
}
